### ISSUE TITLE
Unable to locate package gvfs-bin

### ISSUE THREAD

**Issue by YourTheFinestIveEverKnown** [Zettlr/Zettlr#4761]
### Description

Currently Zettlr V3.0.2 will permanently delete files in MX Linux 23, completely skipping the trash. Okay, fine because the FAQ mentions the need to install gvfs-bin, as referenced below:

https://docs.zettlr.com/en/faq/#im-using-linux-and-deleting-files-doesnt-move-them-to-the-trash

The problem is that gvfs-bin is no longer possible to install (package not located) as it has been deprecated according to Debian:

https://packages.debian.org/buster/gnome/gvfs-bin

*NOTE: The alternative package recommended by Debian is: libglib2.0-bin. Which existed on the system before Zettlr was installed.

### Proposed Changes

Need a way to delete a file that goes to the trash instead of being permanently deleted.

### Caveats

No caveats to my knowledge.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Stable (most recent version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

MX Linux 23.1, KDE

### Additional Information

[CODE]Desktop: KDE Plasma v: 5.27.5 wm: kwin_x11 vt: 7 dm: SDDM Distro: MX-23.1_KDE_x64 Libretto
    July 31 2023 base: Debian GNU/Linux 12 (bookworm)[/CODE]

[CODE]Kernel: 6.1.0-13-rt-amd64 [6.1.55-1] arch: x86_64 bits: 64 compiler: gcc v: 12.2.0[/CODE]

**Comment by nathanlesage** [comment]
There is a specific setting in the advanced preferences to not do that: if moving to trash fails Zettlr will then not permanently delete files. What happens if you switch this setting off? Do you get error messages then? Apparently Electron already switched to a different method five years ago and I just missed the news. I'll be removing that notice from the docs soon.

**Comment by YourTheFinestIveEverKnown** [comment]
Sorry, my mistake. I forgot Dolphin file manager is slow to update and those files I deleted from Zettlr were indeed in the trash. 

Looking in advanced preferences, I do see the setting: 'Delete items irreversibly, if moving them to trash fails'. 
This setting was already unchecked, so I think it's okay. I tested this setting and it does irreversibly delete, so it's fine.

It seems only the documentation needs updating and I was incorrect about files being irreversibly deleted, as Zettlr is working as designed in this regard.
